webapp/tests/Unit/Controller/API/ProblemControllerTest.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types=1);
+
+namespace App\Tests\Unit\Controller\API;
+
+class ProblemControllerTest extends BaseTest
```

Comment:
Maybe add PHPdoc/something that the admin case is explicit handled in another class? Or alternative can you have both classes in the same file, not sure if I'm a fan of that (or if its even possible).